packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -9,19 +9,22 @@ import { Icon, IIconStyles, IStackStyles, ITheme, memoizeFunction, Stack, useThe
  */
 export type NotificationIconProps = {
   chatMessagesCount: number;
+  label?: string;
 };
 /**
  * @private
  */
 export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
-  const { chatMessagesCount } = props;
+  const { chatMessagesCount, label } = props;
   const theme = useTheme();
   const renderNumber = (numberOfMessages): JSX.Element => {
     if (numberOfMessages < 1) {
       return <></>;
     } else {
       return (
-        <Stack styles={notificationTextStyles(theme)}>{numberOfMessages < 9 ? <>{numberOfMessages}</> : <>9+</>}</Stack>
+        <Stack aria-label={label} role={'alert'} styles={notificationTextStyles(theme)}>
```

Comment:
Presumably with `alert` the screen reader only announces it once?

Reply:
with alert it announces whenever there is another new message. do we know if we want to do this? or should I take it out so that it only announces if there is a new message when they enter scan mode?

Reply:
My thinking is when it goes from 0 new messages -> some new messages there should be an alert announcement,
For 1 new messages -> 3 new messages that would be a `status` role.
If there's an alert everytime there's a new message that would be very distracting to someone trying paying attention to a meeting
For alert the docs say 
> Warning: Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role
So we need to be careful with this role